docs/data/tree-view/rich-tree-view/expansion/expansion.md

```diff
@@ -56,9 +56,10 @@ Use the `setItemExpansion` API method to change the expansion of an item.
 apiRef.current.setItemExpansion(
   // The DOM event that triggered the change
   event,
-  // The ID of the item to expand or collapse
+  // The id of the item to expand or collapse
```

Comment:
I tried to unify a bit the wording for those boolean parameters throughout the Tree View codebase
And to always have the same descriptions for the params in the JSDoc and in the doc section